hazelcast/src/main/java/com/hazelcast/partition/impl/ReplicaSyncInfo.java

```diff
@@ -18,7 +18,7 @@
 
 import com.hazelcast.nio.Address;
 
-final class ReplicaSyncInfo {
+public final class ReplicaSyncInfo {
```

Comment:
I thought the same thing. But what if another test, which is not under partition.impl package, wants to use that TestPartitionUtils class?